association_backend/association/api/mutation/__init__.py

```diff
@@ -0,0 +1,7 @@
+from ..builder import create_mutation_type
+from .manage_user_association_subscription import manage_user_association_subscription
+from .subscribe_user_to_association import subscribe_user_to_association
+
+Mutation = create_mutation_type(
+    "Mutation", [subscribe_user_to_association, manage_user_association_subscription]
+)
```

Comment:
remove this from here and move it into a `mutation.py` file

The mutations files will instead be inside a `mutations` directory have a look at my or #2084 to see

Reply:
ok done